docs/src/tutorials/tutorial_4.md

```diff
@@ -0,0 +1,304 @@
+# **Tutorial 4:** Do you know our Mascot? - Learn about transitions and subactions!
+
+You have learned a couple of cool features of Javis already. Now you're ready to finally meet our little mascot. Well actually you can't just see him, we have to create him first. :smile:
+
+Let's create a list of what we want first
+- a circular head
+- some hair
+- eyes
+- a nose
+- a moving mouth
+- he should say something
+
+This should show you the power of `SubAction` so we want each of those appear (fade in) one after another.
```

Comment:
I'd suggest putting this line after you define the title and background so you can better and more clearly introduce a SubAction.

Reply:
I changed this to the learning outcome now so have it still at the top.